nexus/src/authz/omicron.polar

```diff
@@ -240,15 +244,28 @@ resource SiloUser {
 	    "create_child",
 	];
 
+	roles = ["admin", "viewer"];
```

Comment:
This isn't ironclad or anything but we don't bother specifying Oso roles for things where we don't support actually assigning those roles.  That is, Oso roles are 1:1 with roles that might be stored in the database.

You could remove these, plus the rules at L255-L260, and just define some `has_permission` rules.